artifacts/pom.xml

```diff
@@ -373,8 +397,45 @@
                             <version>${certify-mock-plugin.version}</version>
                             <outputDirectory>${certify-plugin.location}</outputDirectory>
                             <destFileName>${certify-mock-plugin.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
+<!-- artifactItem section for certify postgres dataprovider plugin -->
+					<artifactItem>
+						<groupId>io.mosip.certify</groupId>
+						<artifactId>postgres-dataprovider-plugin</artifactId>
+						<version>${certify-postgres-dataprovider-plugin.version}</version>
+						<outputDirectory>${certify-plugin.location}</outputDirectory>
+						<destFileName>${certify-postgres-dataprovider-plugin.fileName}</destFileName>
                             <type>jar</type>
                     </artifactItem>
+<!-- artifactItem section for certify mock ida dataprovider plugin -->
+					<artifactItem>
+						<groupId>io.mosip.certify</groupId>
+						<artifactId>mock-ida-dataprovider-plugin</artifactId>
+						<version>${certify-mock-ida-dataprovider-plugin.version}</version>
+						<outputDirectory>${certify-plugin.location}</outputDirectory>
+						<destFileName>${certify-mock-ida-dataprovider-plugin.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
+
+					<artifactItem>
+						<groupId>io.mosip.esignet.plugin</groupId>
+						<version>${mosip-identity-plugin.version}</version>
+						<artifactId>mosip-identity-plugin</artifactId>
+						<outputDirectory>${esignet-plugins.location}</outputDirectory>
+						<destFileName>${mosip-identity-plugin.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
+
+					<artifactItem>
+						<groupId>io.mosip.esignet.plugin</groupId>
+						<version>${esignet-mock-plugin.version}</version>
+						<artifactId>mock-plugin</artifactId>
+						<outputDirectory>${esignet-plugins.location}</outputDirectory>
+						<destFileName>${esignet-mock-plugin.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
```

Comment:
The above 2 sections is again repeating.

Reply:
Removed repeated entries